core/src/consensus.rs

```diff
@@ -115,6 +120,13 @@ impl Tower {
             }
             let mut vote_state = vote_state.unwrap();
 
+            for vote in &vote_state.votes {
+                lockout_intervals
+                    .entry(vote.expiration_slot())
+                    .or_insert_with(|| vec![])
+                    .push((vote.slot, key));
```

Comment:
Will add these keys to the replay_stage `all_pubkeys` so they pull from the same reference pool to reduce memory usage.